### PR TITLE
runtime: implement System.Runtime.LoadScript, fix #2701

### LINKED ISSUE #2701: Dynamic RO scripts (eval)
Implement neo-project/neo#2756.

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

**Review by AnnaShaleva — COMMENTED** [review]
LGTM for the current stage, I wasn't able to find any bugs connected with context unloading.

File: pkg/vm/context.go [hunk 1]
```diff
@@ -357,3 +361,18 @@ func (c *Context) HasTryBlock() bool {
...
 	return false
 }
+
+// DynamicOnUnload implements OnUnload script for dynamic calls, if no exception
+// has occurred it checks that the context has exactly 0 (in which case a `Null`
+// is pushed) or 1 returned value.
+func DynamicOnUnload(v *VM, ctx *Context, commit bool) error {
+	if commit {
+		eLen := ctx.Estack().Len()
+		if eLen == 0 { // No return value, add one.
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/vm/context.go:371]: The last one usually is an entry script with `-1`, so practically it shouldn't be a problem, but maybe it's time for a little refactor of `RET` handler, these checks are better always be done.

**Comment by roman-khimov** [comment]
Rebased this one as well.

**Review by AnnaShaleva — COMMENTED** [review]
And we need one more commit that updates NNS's neo-go dependency.

**Comment by roman-khimov** [comment]
> And we need one more commit that updates NNS's neo-go dependency.

Do we? Anything changed in neo-go to affect NNS?

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**